### PR TITLE
Support to use SageMaker Containers outside SageMaker training

### PR DESCRIPTION
**Pull request by mvsusp**
*Description of changes:*
- SageMaker containers create a folder under ~/sagemaker_local/jobs with the required directories and files when used outside SageMaker Training 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker_containers/__init__.py [hunk 1]
```diff
@@ -12,15 +12,21 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+from sagemaker_containers import _logging
+
+logger = _logging.get_logger()
```
Comment (ChoiByungWook) [inline src/sagemaker_containers/__init__.py]: ?

File: src/sagemaker_containers/_env.py [hunk 2]
```diff
@@ -84,6 +137,31 @@
...
+    logger.info('Creating a new training folder under %s .' % base_dir)
+
+    os.makedirs(model_dir)
+    os.makedirs(input_config_dir)
+    os.makedirs(output_data_dir)
+
+    _write_json({}, hyperparameters_file_dir)
+    _write_json({}, input_data_config_file_dir)
+
+    host_name = socket.gethostname()
```
Comment (ChoiByungWook) [inline src/sagemaker_containers/_env.py]: Will this work for every docker container?

Comment (mvsusp) [reply src/sagemaker_containers/_env.py]: I believe so. It works for Windows, Mac, and Unix https://docs.python.org/2/library/socket.html

File: src/sagemaker_containers/__init__.py (hunk 1 above)
Comment (mvsusp) [reply src/sagemaker_containers/__init__.py]: Good catch, I will remove it.

**Pull request merged**